src/DateTimePicker.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import cx from 'classnames';
+import Flatpickr from './FlatpickrComponent';
```

Comment:
yeah, probably keep it as `FlatpickrComponent` or event change filenames to make it `ReactFlatpickr` or something - that would potentially be open-sourceable for other devs to use.